recipes/cpptango/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cppTango" %}
+{% set version = "9.3.4" %}
+{% set sha256 = "50828ae9f523c8b2eeb546b38138b0d0882fcd3050e952d93a66a594ed178e72" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tango-controls/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports :
+    - {{ pin_subpackage('cpptango', min_pin='x.x', max_pin='x.x') }}
```

Comment:
Same here:
```suggestion
    - {{ pin_subpackage('cpptango', max_pin='x.x') }}
```

Reply:
Same answer here.
These versions are binary compatible so should be compatible.